arm-sql/2014-04-01/examples/DatabaseCreateImportMax.json

```diff
@@ -0,0 +1,42 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "sqlcrudtest-4799",
+    "serverName": "sqlcrudtest-5961",
+    "databaseName": "testdb",
+    "api-version": "2014-04-01",
+    "parameters": {
+      "databaseName":"TestDbImport",
+      "edition":"Basic",
+      "serviceObjectiveName":"Basic",
+      "maxSizeBytes":"2147483648",
```

Comment:
I'm documenting all the bugs we come across and will bring them up when we discuss API changes